packages/api/game/base.go

```diff
@@ -0,0 +1,44 @@
+package game
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/gorilla/websocket"
+)
+
+func NewGame(ctx context.Context, conn *websocket.Conn) *Game {
+	return &Game{
+		ctx:        ctx,
+		connection: conn,
```

Comment:
context and connections should not be stored like this (really context should never be stored), since they a coupled to the lifecycle of the websocket